projects/bootloader/src/update_name.c

```diff
@@ -0,0 +1,90 @@
+#include "update_name.h"
+
+#include <pb_common.h>
+#include <pb_decode.h>
+#include <pb_encode.h>
+#include <stdio.h>
+
+#include "bootloader_can.h"
+#include "bootloader_datagram_defs.h"
+#include "can_datagram.h"
+#include "config.h"
+#include "crc32.h"
+#include "dispatcher.h"
+#include "log.h"
+#include "reset.h"
+#include "update_name.pb.h"
+
+// Note: Certain functionality will be given to the client script to control
+// The client script will make sure only one board is edited per command and
+// will make sure the id given is not currently used
+
+static UpdateName s_name_proto = UpdateName_init_zero;
```

Comment:
Could this be a local variable? We're trying to keep the executable size to a minimum, and this seems like a decently large struct.